.ci/windows-steps.yaml

```
@@ -118,6 +118,61 @@ steps:
     replaceExistingArchive: true
   displayName: 'Build artifacts'
 
+# Build MSI installer.
+- powershell: |
+    # Pull the documentation for the installer.
+    try {
+      $url = "http://ci.mlpack.org/job/mlpack%20-%20doxygen%20build/lastSuccessfulBuild/artifact/build/doc/html/*zip*/html.zip"
+      (new-object net.webclient).DownloadFile($url, 'dist\win-installer\jenkinsdoc.zip')
+    }
+    catch {
+      Write-Output "Unable to download precompiled Doxygen documentation from Jenkins!"
+    }
+    try {
+      (Add-Type -AssemblyName System.IO.Compression.FileSystem);
+      [System.IO.Compression.ZipFile]::ExtractToDirectory('dist\win-installer\jenkinsdoc.zip', 'dist\win-installer\mlpack-win-installer\Sources\doc')
+    }
+    catch{Write-Output "Unable to add doc to installer, skipping!"}
+    # Preparing installer staging.
+    mkdir dist\win-installer\mlpack-win-installer\Sources\lib
+    cp build\Release\*.lib dist\win-installer\mlpack-win-installer\Sources\lib\
+    cp build\Release\*.exp dist\win-installer\mlpack-win-installer\Sources\lib\
+    cp build\Release\*.dll dist\win-installer\mlpack-win-installer\Sources\
+    cp build\Release\*.exe dist\win-installer\mlpack-win-installer\Sources\
+    cp $(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\bin\x64\*.dll dist\win-installer\mlpack-win-installer\Sources\
+    cp $(Agent.ToolsDirectory)\boost_libs\boost_unit_test_framework-vc*.dll dist\win-installer\mlpack-win-installer\Sources\
```

```suggestion
    cp $(Agent.ToolsDirectory)\boost_libs\boost_unit_test_framework-vc*.dll dist\win-installer\mlpack-win-installer\Sources\
```
I am not sure we still need this line

---

I think you're right---thanks for pointing that out!  I removed it in 1ae1bae7a.  Let's see if everything still works. :)